WalletWasabi.Fluent/Views/AddWallet/WelcomePageView.axaml

```diff
@@ -201,19 +201,18 @@
                   IsDefault="{Binding IsActive}"
                   DockPanel.Dock="Bottom" />
           <TextBlock TextAlignment="Center"
-                     Text="There’s more to discover but it’s better to leave that &#x0a;on you. Without further ado, let’s begin!"
+                     Text="The 1st Rule of Wasabi: Wasabi is for self defense and self defense only"
```

Comment:
This line makes me feel weird. 
Self defense? Like I could harm someone with Wasabi if I used it for not self defense.

Reply:
You can use Wasabi destroy the current fiat-clowncircus by making their fun coupons obsolete but of course no one is doing that 😇

Reply:
You can use Wasabi destroy the current fiat-clowncircus by making their fun coupons obsolete but of course no one is using it for that 😇

Reply:
Not sure ambiguity is bad here at all. It makes people think. Anyway, I'm ok to change it if something better comes along, but for now it's definitely better than the word salad before.